scripts/sysman/ras.yml

```diff
@@ -173,6 +193,27 @@ returns:
         - "Don't have permissions to set thresholds."
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Get the number of RAS error category associated with the error set handle"
```

Comment:
suggest reword "Retrieve number of RAS categories associated with a RAS error handle"

Reply:
Fixed